client/components/post-list-fetcher/index.jsx

```diff
@@ -14,11 +14,12 @@ var PostListStore = require( 'lib/posts/post-list-store' ),
 
 var PostListFetcher;
 
-function dispatchQueryActions( query ) {
-	actions.queryPosts( query );
+function dispatchQueryActions( postListStoreId, query ) {
```

Comment:
Nevermind, I didn't see that this was the fetcher component and that you're specifying a default prop below.